app/assets/javascripts/utilities/updateStyles.js

```diff
@@ -0,0 +1,6 @@
+async function updateStyles(theme) {
+  const css = await fetch(`https://dev.to/assets/themes/${theme}.css`);
```

Comment:
this should take into account that this is not going to run on DEV only, so you'll have to fetch the path without the URL part: `/assets/themes`.

Also, maybe we can call this `updateTheme` so it's a bit easier to find in the future

Reply:
sure will take this in account and makes changes! Thanks!

Reply:
As @rhymes suggested, so this should be good.

```suggestion
async function updateStyles(theme) {
  const css = await fetch(`/assets/themes/${theme}.css`);
```